tests/plugins/test_bump_release.py

```diff
@@ -126,4 +126,5 @@ def getNextRelease(self, build_info):
         parser = DockerfileParser(plugin.workflow.builder.df_path)
         # Only the new-style spellings of the release labels should be set
         assert parser.labels['release'] == next_release
-        assert 'Release' not in parser.labels
+        if 'BZComponent' not in parser.labels.keys():
+            assert 'Release' not in parser.labels
```

Comment:
Can we add the `assert parser.labels['Release'] == next_release` assertion for the other case?